channel/api/snippets/WorkspaceProvisioning/Program.cs

```diff
@@ -0,0 +1,174 @@
+        /*******************************************************************************/
+
+        private static readonly String accountParent = "accounts/" + accountId;
+
+        private static CloudChannelServiceClient cloudChannelServiceClient;
+
+        static void Main(string[] args)
+        {
+            GoogleCredential credential;
+
+            using (var stream = new FileStream(jsonPrivateKeyFile, FileMode.Open, FileAccess.Read))
+            {
+                credential =
+                    GoogleCredential.FromStream(stream)
+                        .CreateScoped(new string[] {"https://www.googleapis.com/auth/apps.order"})
+                        .CreateWithUser(resellerAdminUser);
+            }
+
+            var client =
+                new CloudChannelServiceClientBuilder
+                    {
+                        TokenAccessMethod = ((ITokenAccess) credential).GetAccessTokenForRequestAsync
+                    }
+                    .Build();
+
+            // For the purpose of this codelab, the code lists all offers and picks
+            // the first offer for Google Workspace Business Standard on an Annual
+            // plan. This is needed because offerIds vary from one account to another,
+            // but this is not a recommended model for your production integration
+            PagedEnumerable<ListOffersResponse, Offer> listOffersResponse =
+                client.ListOffers(new ListOffersRequest 
+                {
+                    Parent = accountParent
+                });
+            
+            Offer selectedOffer = new Offer();
+            String sampleOffer = "Google Workspace Business Standard";
+            String samplePlan = "Commitment";
+
+            foreach (Offer offer in listOffersResponse.ToList())
+            {
+                String offerName = offer.Sku.MarketingInfo.DisplayName;
+                String offerPlan = offer.Plan.PaymentPlan.ToString();
+                if (offerName == sampleOffer && offerPlan == samplePlan)
+                {
+                    selectedOffer = offer;
+                    break;
+                }
+            }
+
+            CheckCloudIdentityAccountsExistRequest checkCloudIdentityAccountsExistRequest =
+                new CheckCloudIdentityAccountsExistRequest
+                {
+                    Parent = accountParent,
+                    Domain = customerDomain
+                };
+
+            CheckCloudIdentityAccountsExistResponse checkIdentityResponse =
+                client.CheckCloudIdentityAccountsExist(checkCloudIdentityAccountsExistRequest);
+
+            if (checkIdentityResponse.CloudIdentityAccounts.Count > 0)
+            {
+                throw new Exception(
+                    "Cloud identity already exists. Customer must be transferred. Out of scope for this codelab");
+            }
+
+            PostalAddress postalAddress = new PostalAddress
+            {
+                AddressLines = {
```

Comment:
Here and everywhere, curly brackets on a new line.

```suggestion
                AddressLines = 
                {
```